src/ProgressBar.js

```diff
@@ -89,19 +89,19 @@ class ProgressBar extends React.Component {
     label,
     srOnly,
     striped,
-    active,
+    animated,
     className,
     style,
+    bsStyle,
     ...props
   }) {
     const [bsProps, elementProps] = splitBsProps(props);
-
     const classes = {
       ...getClassSet(bsProps),
-      active,
-      [prefix(bsProps, 'striped')]: active || striped
+      [prefix({ bsClass: 'bg' }, bsStyle)]: bsStyle !== undefined,
+      [prefix(bsProps, 'animated')]: animated,
+      [prefix(bsProps, 'striped')]: animated || striped
```

Comment:
Let's not use the bootstrapUtils here, we are getting rid of a lot of them in #3046 and we can preempt that change by handling the bits manually here like:

```
classes = {
  [bsClass]: true,
  [`bg-${bsStyle}`]: bsStyle,
  [`${bsClass}-animated`]: animated
}
```

We want to not rely on the `splitBsProps` helper as well so we can just spread out the bs props like we do   `animated` or `striped`